src/components/pages/profile/[name]/registration/steps/Profile/profileRecordUtils.ts

```diff
@@ -240,6 +246,8 @@ export const getProfileRecordsDiff = (
     .filter((r) => !!r) as ProfileRecord[]
   const deletedRecords = previousRecords
     .filter((previousRecord) => {
+      if (isEthAddressRecord(previousRecord)) return true
+
```

Comment:
This causing a bug. It will delete any eth value if it exists in the previous record